clippy_lints/src/deprecated_lints.rs

```diff
@@ -4,85 +4,85 @@ macro_rules! declare_deprecated_lint {
     }
 }
 
+declare_deprecated_lint! {
 /// **What it does:** Nothing. This lint has been deprecated.
```

Comment:
Could you also indent the doc comments so that they're aligned with `pub LINT_NAME`?